client/Lykke.Service.Assets.Client/ReadModels/AssetPairsReadModel.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Autofac;
+using Lykke.Service.Assets.Client.Models.v3;
+using Microsoft.Extensions.Caching.Memory;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Lykke.Service.Assets.Client.ReadModels
+{
+    class AssetPairsReadModel : IAssetPairsReadModel, IStartable
```

Comment:
The name sounds like it's the read model object, but it's some kind of repository object. Consider to rename it to AssetParisReadModelRepository